### PR TITLE
added space on build.gradle

### PR DESCRIPTION
**Pull request by Chimakurthi7788**
added space on build.gradle file

### CODE REVIEW

**Review by Chimakurthi7788 — COMMENTED** [review]
here  you need to remove space

**Review by Chimakurthi7788 — COMMENTED** [review]
ok

File: build.gradle [hunk 1]
```diff
@@ -7,7 +7,6 @@ plugins {
   id 'io.spring.javaformat' version '0.0.43'
   id "io.spring.nohttp" version "0.0.11"
 }
-
 apply plugin: 'java'
```
Comment (Chimakurthi7788) [inline build.gradle:10]: hf

**Review by Chimakurthi7788 — COMMENTED** [review]
bgzf

**Review by andrzejsydor — CHANGES_REQUESTED** [review]
What the change is about?

**Pull request closed without merging**